src/windows-manager.cpp

```diff
@@ -1219,6 +1219,21 @@ namespace graphics {
         return true;
     }
 
+    bool WindowsManager::setAlpha(const char *nodeNameCorba, const value_type* integerValue)
+    {
```

Comment:
- Your declaration and definition of `WindowsManager::setAlpha` does not have the same prototype.
- If the value is an integer, please pass an `int`.